src/ofc.ts

```diff
@@ -158,7 +155,7 @@ export function ofcerc20(
   suffix: string = name.replace(/^ofc/, '').toUpperCase(),
   network: OfcNetwork = Networks.main.ofc,
   isToken: boolean = true,
-  addressCoin: string = 'eth',
+  addressCoin: string = 'eth'
```

Comment:
Don't remove the comma dangle! Comma dangles are awesome!

Reply:
I actually added some, but `yarn fmt` doesn't like them

Reply:
> Don't remove the comma dangle! Comma dangles are awesome!

They aren't legal to have there in es5 (end of arguments list). If we want to, we could change this to all, since typescript can handle these.

https://prettier.io/docs/en/options.html#trailing-commas